.gitignore

```diff
@@ -0,0 +1,6 @@
+coverage*.out
+.idea/*
+dist/*
+.DS_Store
+.vscode/
```

Comment:
`.vscode/`, `.idea/*` and `.DS_Store` are not specific to the repo and should be in people's personal gitignores.

Reply:
Good point, but is there any harm in leaving them here? I don't think we have any guides currently for setting up personal .gitignores

Reply:
Happy to remove these once we have a reference guide for setting up personal .gitignores and circulate with the team/ wider eng org?

Reply:
I don't think it's a good idea to pollute the repo `.gitignore`. Probably best to start as we intend to carry on.

Reply:
`man gitignore`

Reply:
Sure, fixed in https://github.com/anz-bank/sysl-go/pull/14/commits/2f01df7b6700f008fa2e8b9114dcdeaeb754ff47